### ISSUE TITLE
Examples for Brats2020 dataset

### ISSUE THREAD

**Issue by dyh127** [MIC-DKFZ/nnUNet#613]
Hi,

Thank you for your excellent work. However, when I try to reproduce your code in Brats2020, I don't know how to train and evaluate it. Even, I cannot find the training and test configures of this dataset.  So would you like to give an introduction about how to preprocess, train and evaluate the Brats2020 dataset?

Best Regards,
Yuhang

**Comment by FabianIsensee** [comment]
Have you seen this:
https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunet/dataset_conversion/Task082_BraTS_2020.py
?

**Comment by dyh127** [comment]
Hi, 

Thank you for your reply. It helps a lot. But there are something I still want to figure out. Would you mind to show the architecture and the training loss for the Brats2020? Thank you so much. 

By the way, there is another minor question about brats challenge dataset. How do you evaluate the validation dataset of Brats2018 and Brats2020? Although this question is not related to your code, I tried find the way for evaluation for a couple of weeks, but I still do not know how to do it. So would you like to teach me? 

Best Regards,
Yuhang

**Comment by FabianIsensee** [comment]
for validation set results you need to upload your images to https://ipp.cbica.upenn.edu/
remember to convert the labels back to brats convention

**Comment by FabianIsensee** [comment]
install hiddenlayer, run the training and you will get a pdf of the architecture in the training output directory

**Comment by dyh127** [comment]
Thank you so much ! I have tested my method in Brats2018 successfully, but when it comes to Brats2020, the following error occurs but I do not know what it means. Have you meet this error? and would you like to share the way to fix it?
<img width="691" alt="error" src="https://user-images.githubusercontent.com/29748305/116786410-03964a80-aad1-11eb-820c-f418d9fecfdb.png">

**Comment by FabianIsensee** [comment]
This is where you should contact IPP support, not nnU-Net :-)

**Comment by dyh127** [comment]
sure, thank you so much for your help.
